sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs

```diff
@@ -128,13 +140,24 @@ public class TestConfigurations
         public static TenantConfiguration DefaultTargetManagedDiskTenant =>
             GetTenant("TargetManagedDiskTenant", s_configurations.Value.TargetManagedDiskTenantName);
 
-
         /// <summary>
         /// Gets a tenant to use for any tests related to blob or container soft delete.
         /// </summary>
         public static TenantConfiguration DefaultTargetSoftDeleteTenant =>
             GetTenant("TargetBlobAndContainerSoftDeleteTenant", s_configurations.Value.TargetSoftDeleteTenantName);
 
+        /// <summary>
+        /// Gets the source tenant to use for any tests related to Object Replication.
+        /// </summary>
+        public static TenantConfiguration DefaultObjectReplicationSourceTenant =>
+            GetTenant("TargetObjectReplicationSourceTenant", s_configurations.Value.TargetOrSourceTenantName);
+
+        /// <summary>
+        /// Gets the source tenant to use for any tests related to Object Replication.
+        /// </summary>
+        public static TenantConfiguration DefaultObjectReplicationDestinationTenant =>
```

Comment:
I thought we were going to take a dependency on the management plane SDK to create ORS policies and assign them, instead of adding a new TenantConfiguration.